### DISCUSSION TITLE
ScriptReference/Callbacks.PostProcessBuildAttribute

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#570] [Notes]
# ScriptReference/Callbacks.PostProcessBuildAttribute

https://docs.unity3d.com/ScriptReference/Callbacks.PostProcessBuildAttribute.html

**Comment by mvi** [comment]
See also https://docs.unity3d.com/ScriptReference/Build.IPostprocessBuildWithReport.html

**Comment by SoylentGraham** [comment]
Note, this attribute has no effect if you're using `IPostprocessBuildWithReport` where `public int callbackOrder { get { return 1; } }` takes precedent
